### ISSUE TITLE
per-byte debugging in _parse11 is excessive

### ISSUE THREAD

**Issue by glennmatthews** [ncclient/ncclient#265]
Reading ncclient's debug logs with NETCONF 1.1 is difficult because of the excessive verbosity of logging literally each single byte read from the session:

```
   ncclient.transport.session: DEBUG: <SSHSession(session, initial daemon)> created: client_capabilities=<dict_keyiterator object at 0x10bed21d8>
   ncclient.transport.ssh: DEBUG: parsing netconf v1.1
   ncclient.transport.ssh: DEBUG: x: b'\n'
   ncclient.transport.ssh: DEBUG: x: b'#'
   ncclient.transport.ssh: DEBUG: x: b'3'
   ncclient.transport.ssh: DEBUG: x: b'0'
   ncclient.transport.ssh: DEBUG: x: b'\n'
   ncclient.transport.ssh: DEBUG: response length: 30
   ncclient.transport.ssh: DEBUG: x: b'<'
   ncclient.transport.ssh: DEBUG: x: b'r'
   ncclient.transport.ssh: DEBUG: x: b'p'
   ncclient.transport.ssh: DEBUG: x: b'c'
   ncclient.transport.ssh: DEBUG: x: b'-'
   ncclient.transport.ssh: DEBUG: x: b'r'
   ncclient.transport.ssh: DEBUG: x: b'e'
   ncclient.transport.ssh: DEBUG: x: b'p'
   ncclient.transport.ssh: DEBUG: x: b'l'
   ncclient.transport.ssh: DEBUG: x: b'y'
   ncclient.transport.ssh: DEBUG: x: b' '
   ncclient.transport.ssh: DEBUG: x: b'x'
   ncclient.transport.ssh: DEBUG: x: b'm'
   ncclient.transport.ssh: DEBUG: x: b'l'
   ncclient.transport.ssh: DEBUG: x: b'n'
   ncclient.transport.ssh: DEBUG: x: b's'
   ncclient.transport.ssh: DEBUG: x: b':'
   ncclient.transport.ssh: DEBUG: x: b'j'
   ...
```

Can we remove the following line, or at least provide an option to disable it while keeping the other debug-level logs available?

```diff
        while True:
            x = buf.read(1)
            if not x:
                logger.debug('No more data to read')
                # Store the current chunk to the message list
                chunk = b''.join(chunk_list)
                message_list.append(textify(chunk))
                break # done reading
-           logger.debug('x: %s', x)
            if state == idle:
                if x == b'\n':
                    state = instart
                    inendpos = 1
```

**Comment by glennmatthews** [comment]
Ah, I see #260 includes this change. Thanks!

**Comment by glennmatthews** [comment]
fixed by #260.
